import_ldraw.py

```diff
@@ -928,6 +964,41 @@ def getColorValue(line, value):
     if value in line:
         n = line.index(value)
         return line[n + 1]
+        
+def linkedParts():
+    """Clean-up design by linking identical parts (mesh/color)"""
+    colors = []
+    for color in bpy.data.materials:
+        colors.append(color.name)
+    
+    parts = []
```

Comment:
Both of these functions need commenting. Why are taking color names and storing them? What happens if ` part.type == 'MESH'` and some part of the part name not in parts? Why are we calling `replaceParts()` and what is going on in that function?